include/geolib/HeightMapNode.h

```diff
@@ -16,6 +16,8 @@ class HeightMapNode {
 
     HeightMapNode(const HeightMapNode& orig);
 
+    HeightMapNode(HeightMapNode&& orig);
```

Comment:
Add a docstring please.

Reply:
This is a move constructor, not very exciting